homeassistant/components/iometer/const.py

```diff
@@ -0,0 +1,7 @@
+"""Constants for the IOmeter integration."""
+
+from datetime import timedelta
+from typing import Final
+
+DOMAIN: Final = "iometer"
+DEFAULT_SCAN_INTERVAL: Final = timedelta(seconds=10)
```

Comment:
`DEFAULT_SCAN_INTERVAL` is only used in the coordinator module. lets move it there